pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -53,6 +53,7 @@ func newPrometheusConverter() *prometheusConverter {
 
 // fromMetrics converts pmetric.Metrics to Prometheus remote write format.
 func (c *prometheusConverter) fromMetrics(md pmetric.Metrics, settings Settings) (errs error) {
+	metricNamer := otlptranslator.MetricNamer{WithMetricSuffixes: settings.AddMetricSuffixes, Namespace: settings.Namespace}
```

Comment:
again, is metricNamer always going to be the same once prometheusConverter is instantiated?